public/index.ts

```diff
@@ -1283,7 +1283,9 @@ function main() {
   } else if (loadTestData) {
     //fetchOpenCell();
     //fetchTiff("https://animatedcell-test-data.s3.us-west-2.amazonaws.com/AICS-12_881.ome.tif", 0);
-    fetchZarr("https://animatedcell-test-data.s3.us-west-2.amazonaws.com/Lamin_multi-06-Deskew-28.zarr", "Image_0", 0);
+    //fetchZarr("https://animatedcell-test-data.s3.us-west-2.amazonaws.com/Lamin_multi-06-Deskew-28.zarr", "Image_0", 0);
```

Comment:
Things we know we shouldn't do, but are much to useful to NOT do.

Reply:
At this point it would be nicer to just stash these as an array of urls and let the code select one rather than using commenting like this.  Maybe that would look friendlier.

Reply:
I added #81 to improve this soon.